test/integration/read_notifications_test.rb

```diff
@@ -26,4 +26,22 @@ class ReadNotificationsTest < ActionDispatch::IntegrationTest
 
     assert_select "h1", "Read notifications"
   end
+
+  test "you can read all unread notifications" do
+    user = users(:with_business)
+    developer = developers(:available)
+    sign_in user
+    conversation = Conversation.create!(developer:, business: user.business)
+    ["Hello!", "Available", "Let's talk"].each do |text|
+      Message.create!(conversation:, sender: developer, body: text)
```

Comment:
Are they any already existing unread notifications from fixtures that we can use?

Reply:
Looks like the features are not adding any notifications.